docs/src/components/Code/Code.js

```diff
@@ -25,16 +25,16 @@ const Code = ({ children, className, live = true, title, ...rest }) => {
 
   const [editorCode, setEditorCode] = useState(children.trim());
 
-  const handleEditorToggle = () => {
+  function handleEditorToggle() {
     // TODO: Make cursor focus on editor when opening
     // should also update styles for focus on editor so it is easier to see that it
     // is actually editable
     setEditorOpen(!editorOpen);
-  };
+  }
 
-  const handleCodeChange = (code) => {
+  function handleCodeChange(code) {
     setEditorCode(code.trim());
-  };
+  }
```

Comment:
@lordpixel did you decide to change this, or did the linter do this? I am worried seeing this because we prefer arrow function syntax.

Reply:
Should be an arrow func. I am looking into why the rule is allowing this.

Reply:
yes @brianespinosa, we've been fixing these since we started solving eslint errors.

Reply:
@brianespinosa this was not my decision, there is a rule that is enforcing this... 
![image](https://user-images.githubusercontent.com/13924868/110031429-83d04780-7cfc-11eb-906a-7893a23033f4.png)